packages/server-web/src/client/component/Login.vue

```diff
@@ -1,9 +1,12 @@
 <template>
   <div class="todasticapp">
     <h1>Todastic</h1>
-    <input type="text" name="username" v-model="input.username" placeholder="Username" />
-    <input type="password" name="password" v-model="input.password" v-on:keydown.enter="login()" placeholder="Password" />
-    <button type="button" v-on:click="login()">Login</button>
+    <span></span> <!-- quickfix for grid layout -->
```

Comment:
Are you sure there is no better way?

Reply:
There is a better way, for sure. I think we have to revisit the layout soon anyway (especially the scripting input looks not really good). That's why i thought a hacky quickfix is ok for now.